sql/api/src/main/scala/org/apache/spark/sql/types/TimeType.scala

```diff
@@ -52,6 +52,7 @@ object TimeType {
   val MICROS_PRECISION: Int = 6
   val MAX_PRECISION: Int = MICROS_PRECISION
```

Comment:
Is `MAX_PRECISION` equals `NANOS_PRECISION` ?

Reply:
Not yet. From user perspective, Spark SQL supports time precisions up to microseconds so far.

Reply:
Get it.